doc/observability-deployment.adoc

```diff
@@ -18,7 +18,13 @@ Retrieve available Kibana dashboards tuned for Open Liberty logging events link:
 
```

Comment:
Retrieve available Kibana dashboards tuned for Open Liberty logging events
->
Retrieve available Kibana dashboards for displaying Open Liberty logging events